packages/default-reporter/src/reporterForClient/reportProgress.ts

```diff
@@ -30,10 +34,11 @@ export default (
     ? throttledProgressOutput.bind(null, opts.throttleProgress)
     : nonThrottledProgressOutput
 
-  return getModulesInstallProgress$(log$.stage, log$.progress)
-    .map(({ importingDone$, progress$, requirer }) => {
-      const output$ = progressOutput(importingDone$, progress$)
-
+  return getModulesInstallProgress$(log$.stage, log$.progress, log$.importing)
+    .map(({ importingDone$, progress$, importingMethod$, requirer }) => {
+      const nodeModulesPath = path.join(requirer, 'node_modules')
+      const showImportingMethod = !fs.existsSync(nodeModulesPath)
```

Comment:
this will not always work because we can set a custom modules directory with the modules-dir setting. We get the settings passed to the reporter, so this option should be available